docs/usage.md

```diff
@@ -43,10 +43,10 @@ The typical commands for running the pipeline on AF2 and Colabfold modes are as
 nextflow run nf-core/proteinfold \
        --input samplesheet.csv \
        --outdir <OUTDIR> \
-       --mode AF2 \
-       --af2_db <null (default) | DB_PATH> \
+       --mode alphafold2 \
```

Comment:
A reminder of my first comment in the README regarding the command for AF2 split

Reply:
See here [5d99cff](https://github.com/nf-core/proteinfold/pull/61/commits/5d99cff576fe81028b0e1696a7213172ed6c4382)